cmd/graph.go

```diff
@@ -15,7 +15,7 @@ func cmdGraph(cmd *cli.Cmd) {
 	defaultOpts(&opts, cmd)
 
 	opts.dryRun = true
-	opts.generateGraph = true
+	opts.disableGenerateGraph = false
```

Comment:
You can remove this line as the default value for an uninitialized boolean is `false`